lib/index.d.ts

```diff
@@ -0,0 +1,6 @@
+declare function sort(array: unknown[], opts: Options): any[];
```

Comment:
Shouldn't this be defined with generic item?

Reply:
I think you are right. A generic would connect the input type with the output type.